src/assets/styles/app.css

```diff
@@ -0,0 +1,271 @@
+    padding: 0px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    font-style:italic;
+    padding: 0px;
+    width: 80px;
+}
+
+.logout-button {
+    position:absolute;
+    top:0;
+    right:0;
+    background-color: #bc4749;
+    border: 10px solid #bc4749;
+    border-radius: 0px;
+    color: black;
+    padding: 0px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    font-style:italic;
+    padding: 0px;
+    width: 80px;
+}
+
+.login-button {
+    position:absolute;
+    top:0;
+    right:100px;
+    background-color: #6a994e;
+    border: 10px solid #6a994e;
+    border-radius: 0px;
+    color: black;
+    padding: 0px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    font-style:italic;
+    padding: 0px;
+    width: 80px;
+}
+
+.register-button:hover{
+    background-color: #d1dbe6;
+    border-color: #d1dbe6;
+}
+.logout-button:hover{
+    background-color: #d28586;
+    border-color: #d28586;
+}
+.login-button:hover{
+    background-color: #9fc389;
+    border-color:  #9fc389;
+}
+input[type="text"], input[type="password"], input[type="email"]{
+    margin: 20px 0;
+    width: 100%;
+    padding: 10px;
+}
+textarea{
+    margin: 20px 0;
+    width: 100%;
+    padding: 10px;
+    height: 100px;
+    resize: vertical;
+}
+label {
+    font-size: 26px;
+    display:block;
+    clear:both;
+}
+label[for="registration_form_agreeTerms"] {
+    font-size: 16px;
+}
+
+input:hover {
+    box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.94);
+}
+button[type="submit"] {
+    background-color: #6a994e;
+    width: 180px;
+    display: block;
+    margin: 0px auto;
+    border: none;
+    border-radius: 5px;
+    font-weight: bold;
+    font-size: 18px;
+    font-family: "Courier New", Courier, monospace;
+    color: white;
+    margin-top: 20px;
+    padding: 20px;
+    text-align: center;
+    cursor: pointer;
+}
+
+button[type="submit"]:hover {
+    background-color: #9fc389;
+}
+.login-form-container {
+    width: 40%;
+    margin: 0 auto;
+}
+.regisration-form-container {
+    width: 40%;
+    margin: 0 auto;
+}
+.creation-form-container {
+        width: 40%;
+        margin: 0 auto;
+}
+.imgmain {
+    display: block;
+    border: 4px solid var(--color1);
+    border-radius: 2%;
+    padding: 0;
+    text-align: center;
+    width: 500px;
+    height: 100%;
+    margin-left: auto;
+    margin-right: auto;
+  }
+  
+  .imgmain img {
+    display: block;
+    width: 100%;
+    height: auto;
+    max-width: 100%;
+  }
+  .Difficultyimg {
+    margin: 0;
+    padding: 0;
+    max-width: 50px;
+    max-height: 50px;
+    padding-right: 10px;
+    padding-left: 58px;
+  }
+  
+  .Difficulty {
+    font-size: 50px;
+    max-width: 50px;
+    max-height: 50px;
+    display: flex;
+    align-self: center;
+  }
+
+  .form-recenze-container {
+    width: 40%;
+    margin: 0 auto;
+  }
```

Comment:
Also, the selectors should ideally use consistent naming (so not like `.Someselector` in some cases and `.some-selector` in other)